README.md

```diff
@@ -229,19 +341,52 @@ missing some functionality and are therefore not supported. You have to clone th
 repository manually and use the `Reopen in Container` or `Open Folder in Container...`
 command.
 
-## Architecture
+## Running Tests
+
+Once you get the [development environment for a particular ecosystem](TODO:hyperlink) going, execute the tests for that ecosystem by
+running `rspec spec` inside that ecosystem's folder, e.g.
+
+```shell
+$ cd go_modules
+$ bundle exec rspec spec
+```
+
+You can also limit the tests to only the file you're working on, or only tests that previously failed, for example:
+
+```shell
+$ rspec spec/dependabot/file_updaters/elixir --only-failures
+```
 
-Dependabot Core is a collection of Ruby packages (gems), which contain the
+Style is enforced by RuboCop. To check for style violations, simply run `rubocop` in
+each of the packages, e.g.
+
+```shell
+$ cd go_modules
+$ bundle exec rubocop
```

Comment:
In the containers, there's a single version installed for each gem. So I think we can simplify this and the above commands to remove `bundle exec`.